src/Blazor.Auth0.ClientSide/AuthenticationService.cs

```diff
@@ -77,16 +77,15 @@ private set
         /// <param name="uriHelper">A <see cref="IUriHelper"/> param.</param>
         /// <param name="options">A <see cref="ClientOptions"/> param.</param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1201:Elements should appear in the correct order", Justification = "I like this best ;)")]
-        public AuthenticationService(ILogger<AuthenticationService> logger, IComponentContext componentContext, HttpClient httpClient, IJSRuntime jsRuntime, IUriHelper uriHelper, ClientOptions options)
+        public AuthenticationService(ILogger<AuthenticationService> logger,  HttpClient httpClient, IJSRuntime jsRuntime, NavigationManager uriHelper, ClientOptions options)
```

Comment:
Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?